### PR TITLE
Add Estonia (EE) playbook

### LINKED ISSUE #91: [Playbook]: Estonia (EE)
### Country

Estonia (EE)

### eAIP

_No response_

### Steps to add a new playbook

You can check [playbooks](https://github.com/drph4nt0m/avbot-charts/tree/main/playbooks) for examples on chart playbooks and [axios engine](https://github.com/drph4nt0m/avbot-charts/blob/main/app/engines/axios.js) on how the features in the playbook are used by the axios scraping engine.

Tasks summary:
-[ ] Fork & clone AvBot Charts
-[ ] Create playbook for the country in playbooks
-[ ] Add tests for the country in tests/getChart.test.js (tests should be in alphabetical order of the country code)
-[ ] Push changes and submit a pull request
-[ ] If you need any help, reach out to us on our [Discord server](http://link.avbot.in/support).

Are you ready to work on this issue? Let us know, and we will assign it to you.

Happy Coding!

### PR DESCRIPTION
**Pull request by Matrix278**
# Description

Adds Estonia (EE) playbook.

Fixes #91 

## How Has This Been Tested

Added new tests for the country in tests/getChart.test.js

## Checklist

- [x] My code follows the style guidelines of this project
- [x] I have performed a self-review of my own code
- [ ] I have commented my code, particularly in hard-to-understand areas
- [ ] I have made corresponding changes to the documentation/readme
- [x] My changes generate no new warnings
- [ ] Any dependent changes have been merged and published in downstream modules

### CODE REVIEW

File: tests/getChart.test.js [hunk 1]
```diff
@@ -62,6 +62,16 @@ describe('CR', () => {
   });
 });
 
+describe('EE', () => {
+  test('Found', async () => {
+    expect(await getChartWrapper('EE', 'GEN-1.7')).toMatch(uriRegEx);
```
Comment (akgupta0777) [inline tests/getChart.test.js:67]: GEN-1.7 is not an airport or aerodrome. it's heading link from that website.

File: playbooks/EE.json [hunk 2]
```diff
@@ -0,0 +1,33 @@
...
+            {
+              "selector": "frame[name=\"eAISNavigation\"]",
+              "attribute": "src"
+            }
+          ]
+        }
+      ],
+      "chart": {
+        "baseUrl": "${baseUrl}/eAIP/",
+        "selector": "a[id=\"${icao}\"]",
```
Comment (akgupta0777) [inline playbooks/EE.json:28]: This is not a right way to get selector according to EE eAIP.

**Review by akgupta0777 — COMMENTED** [review]
Make required changes and push again.

**Comment by Matrix278** [comment]
Changed, is it right now?

**Review by akgupta0777 — APPROVED** [review]
Seems good to me

**Review by drph4nt0m — APPROVED** [review]

**Comment by drph4nt0m** [comment]
@Matrix278 thanks for your contrinbution. Looking forward to more PRs from you.

**Pull request merged**